docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -136,12 +141,11 @@ var x = $"""
     """;
 ```
 
-
 ## Inferred delegate type for methods includes default parameter values and `params` modifier
 
 ***Introduced in Visual Studio 2022 version 17.5***
 
-In .NET SDK 7.0.100 or earlier, delegate types inferred from methods ignored default parameter values and `params` modifiers
+In .NET SDK 7.0.100 or earlier, `delegate` types inferred from methods ignored default parameter values and `params` modifiers
```

Comment:
same:

```suggestion
In .NET SDK 7.0.100 or earlier, delegate types inferred from methods ignored default parameter values and `params` modifiers
```

Reply:
I may have done this as a larger search and replace effort so it may appear several times... :|